src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -79,6 +88,10 @@ private void updateOutputLists()
          _allAvailableOutputs.Clear();
          var outputs = _entitiesInSimulationRetriever.OutputsFrom(_simulation);
          _allAvailableOutputs.AddRange(outputs.Select(x => mapFrom(_simulation, x)).OrderBy(x => x.DisplayString));
+
+         //adding a none DTO output, in order to be able to select it and delete the outputMapping
+         var noneEntry = new SimulationQuantitySelectionDTO(_simulation, null, Captions.SimulationUI.NoneEditorNullText);
```

Comment:
I tnink this should be a member of the presenter so that you can reuse it later instead of a local variable

Reply:
done